src/components/chart.tsx

```diff
@@ -506,17 +510,21 @@ const BooleanChart: FunctionComponent<ChartProps> = ({
         isOverflowingLeft && overflowLeftStyle,
         isOverflowingRight && overflowRightStyle,
       )}
-      ref={elementRef}
-      onScroll={recomputeScrolling}
     >
-      <div class={innerBooleanChartStyle}>
-        {points.map((p, i) => (
-          <BooleanDisplay
-            key={i} // eslint-disable-line caleb/react/no-array-index-key
-            value={Boolean(p)}
-            onClick={() => onPointClick(i)}
-          />
-        ))}
+      <div
+        class={innerBooleanChartWrapperStyle}
+        ref={elementRef}
+        onScroll={recomputeScrolling}
+      >
+        <div class={innerBooleanChartStyle}>
+          {points.map((p, i) => (
+            <BooleanDisplay
+              key={i} // eslint-disable-line caleb/react/no-array-index-key
```

Comment:
Isn't it not great to use the index key since React won't be able to uniquely identify elements?  I guess it doesn't matter too much since once they're rendered we never change/re-order them (like sort or w/e). Plus it's not introduced via this PR. Maybe a comment clarifying why this is fine would be good?